__tests__/BookmarkRedux.test.js

```diff
@@ -0,0 +1,112 @@
+//https://redux.js.org/usage/writing-tests -> Unit Testing Individual Functions
+
+import reducer, {
+  addBookmark,
+  removeBookmark,
+} from "../src/Redux/bookmarkSlice.js";
+
+describe("Bookmark Redux test list of bookmark items", () => {
```

Comment:
Good job on figuring out how to do the unit testing!

Reply:
Thanks!  It's a lot easier than doing the component testing.